docs/pages/release_notes.md

```diff
@@ -60,6 +60,8 @@ This is a {{ site.pmd.release_type }} release.
     *   [#1542](https://github.com/pmd/pmd/pull/1542): \[apex] Include the documentation category
     *   [#1546](https://github.com/pmd/pmd/issues/1546): \[apex] PMD parsing exception for Apex classes using 'inherited sharing' keyword
     *   [#1568](https://github.com/pmd/pmd/pull/1568): \[apex] AST node attribute @Image not usable / always null in XPath rule / Designer
+*   cpp
+    *   [#1559](https://github.com/pmd/pmd/issues/1559): \[cpp] CPD: Lexical error in file (no file name provided)
```

Comment:
I think we should move this to core / all. Even if the original bug was filed for cpp, this affects all languages.